pulpcore/cli/container/repository.py

```diff
@@ -37,6 +38,16 @@
 
 translation = get_translation(__name__)
 _ = translation.gettext
+VALID_TAG_REGEX = r"^[A-Za-z0-9][A-Za-z0-9._-]*$"
+
+
+def _tag_callback(ctx: click.Context, param: click.Parameter, value: str) -> str:
+    if len(value) == 0:
+        raise click.ClickException("Please pass a non empty tag name.")
```

Comment:
FWIW, an empty string would not pass the regex. But i see this check provide a better error condition.